Tests/Clappr_Tests/Classes/Plugin/Playback/AVFoundationPlaybackQualityMetricsTests.swift

```diff
@@ -0,0 +1,117 @@
+import Quick
+import Nimble
+import OHHTTPStubs
+
+@testable import Clappr
+
+class AVFoundationPlaybackQualityMetricsTests: QuickSpec {
+    override func spec() {
+        describe(".AVFoundationPlaybackQualityMetricsTests") {
```

Comment:
Are these tests supposed to be integration ones?

Reply:
No, we've changed the header from response and now the stub are working. Could you check, please?

Reply:
So maybe you should call the `onAccessLogEntry` method directly and config some spy object to catch the event sent.

Reply:
Nice =]
Now we are checking with a spy object the event sent, the other tests don't have a `kSourceURL`.